doc/contribute/contributor_expectations.rst

```diff
@@ -305,10 +305,23 @@ Reviewer Expectations
   they address all non-blocking comments. PR authors should acknowledge every
   review comment in some way, even if it's just with an emoticon.
 
+- Style changes that the reviewer disagrees with but that are not documented as
+  part of the project can be pointed out as non-blocking, but cannot constitute
+  a reason for a request for changes. The reviewer can optionally correct any
+  potential inconsistencies in the tree, document the new guidelines or rules,
+  and then enforce them as part of the review.
+
+- Whenever requesting changes, reviewers must be able to point out the
```

Comment:
Shouldn't this say "style-related changes"? I don't see how we could ever provide guidelines for every kind of change that a reviewer/maintainer might request, especially technical ones, design/architecture-related, choice of algorithms, etc.

Reply:
Yes, agreed, I kind of hesitated here myself. I wonder what @nashif @kartben and others think as well.

Reply:
yes, agree. Need to distinguish between change requests about the code, design and and logic vs how they were written and the style used.